nix/runtime.nix

```diff
@@ -52,6 +52,15 @@ rec {
         filter = builtins.toJSON { const = true; };
       };
     };
+    kup = {
```

Comment:
Why is it not `kupo`?

Reply:
You typed `kup` there in our brief meeting and I assumed there was a reason for that (lol)